scripts/Readme.md

```diff
@@ -0,0 +1,33 @@
+# Building DTK on ALCF ressources
+
+## Software environment
+The MPICH wrappers of the GNU Compiler Collection (GCC) were used to build DTK on Mira/Cetus.
+Below is the corresponding SoftEnv configuration file `$HOME/.soft.mira`.
```

Comment:
Is .soft.mira a required name for the file? Is it something specific for ALCF or just the way you named the file?